db/seeds.rb

```diff
@@ -6,24 +6,24 @@
+item2 = brewing.items.create!(title: 'Columbian Dark Coffee', description: 'Creamy and smooth-bodied with a smoky aroma and well-balanced flavor.', price: 1500, image: File.new("#{Rails.root}/app/assets/images/coffee.bag.jpg"))
 
 item3 = merchandise.items.create!(title: 'Clear Kit', description: 'Full Kit.', price: 8000, image: File.new("#{Rails.root}/app/assets/images/merch.jpg"))
-item4 = merchandise.items.create!(title: 'Coffee Grinder', description: 'Classic hand crank coffee grinder', price: 2000, image: File.new("#{Rails.root}/app/assets/images/grinder.jpg"))
-item5 = merchandise.items.create!(title: 'Espresso Machine', description: 'Bring out the indulgent European in you', price: 12000, image: File.new("#{Rails.root}/app/assets/images/espresso-machine.jpg"))
-item6 = merchandise.items.create!(title: 'Coffee Maker', description: 'Chrome coffee maker, for the baller in you', price: 5000, image: File.new("#{Rails.root}/app/assets/images/coffee-maker.jpg"))
+item4 = merchandise.items.create!(title: 'Coffee Grinder', description: 'Classic hand crank coffee grinder.', price: 2000, image: File.new("#{Rails.root}/app/assets/images/grinder.jpg"))
+item5 = merchandise.items.create!(title: 'Espresso Machine', description: 'Bring out the indulgent European in you.', price: 12000, image: File.new("#{Rails.root}/app/assets/images/espresso-machine.jpg"))
+item6 = merchandise.items.create!(title: 'Coffee Maker', description: 'Chrome coffee maker, for the baller in you.', price: 5000, image: File.new("#{Rails.root}/app/assets/images/coffee-maker.jpg"))
 
 item7 = chow.items.create!(title: 'Artichoke Souffle', description: 'Try this rich-tasting cheese, artichoke and spinach soufflé recipe for your next brunch.', price: 700, image: File.new("#{Rails.root}/app/assets/images/breakfast.jpg"))
 item8 = chow.items.create!(title: 'Berry Tart', description: 'Lightly sweetened berries top vanilla bean-flecked pastry cream contained in a graham cracker crust.', price: 500, image: File.new("#{Rails.root}/app/assets/images/tarts.jpg"))
 item9 = chow.items.create!(title: 'Caprese Panini', description: 'The classic combination of mozzarella, tomatoes and basil is known as caprese.', price: 700, image: File.new("#{Rails.root}/app/assets/images/panini.jpg"))
-item10 = chow.items.create!(title: 'Blueberry Muffin', description: 'blueberry muffins balance a moist, buttery crumb topping and are equally delicious with blueberries in season.', price: 400, image: File.new("#{Rails.root}/app/assets/images/muffin.jpg"))
+item10 = chow.items.create!(title: 'Blueberry Muffin', description: 'Blueberry muffins balance a moist, buttery crumb topping and are equally delicious with blueberries in season.', price: 400, image: File.new("#{Rails.root}/app/assets/images/muffin.jpg"))
```

Comment:
Useless assignment to variable - `item10`.<br>Line is too long. [256/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.